ods_ci/tasks/Tasks/patch_dsc_cr_operatorv2.robot

```diff
@@ -0,0 +1,185 @@
+*** Settings ***
+Documentation    Tasks related to the operator rearchitecture
+Library          String
+Library          OperatingSystem
+Library          ../../libs/Helpers.py
+
+
+*** Variables ***
+@{COMPONENTS} =    dashboard    datasciencepipelines    distributedWorkloads    kserve    modelmeshserving    workbenches  # robocop: disable
+${DSC_NAME} =    default
+${PATCH_PREFIX} =    oc patch datasciencecluster ${DSC_NAME} --type='merge' -p '{"spec": {"components": {
+
+
+
+*** Tasks ***
+# Patch DataScienceCluster CustomResource
+#     [Documentation]
+#     Log to Console    Requested Configuration:
+#     FOR    ${cmp}    IN    @{COMPONENTS}
+#         Log To Console    ${cmp} - ${COMPONENTS.${cmp}}
+#     END
+#     Patch DataScienceCluster CustomResource Using Test Variables
+#     FOR    ${cmp}    IN    @{COMPONENTS}
+#         IF    "${COMPONENTS.${cmp}}" == "True"
+#             Component Should Be Enabled    ${cmp}
+#         ELSE IF    "${COMPONENTS.${cmp}}" == "False"
+#             Component Should Not Be Enabled    ${cmp}
+#         ELSE
+#             Fail    msg=Invalid parameters in test-variables.yml
+#         END
+#     END
+
+# Switch To Core Installation Profile
```

Comment:
Could we remove profile text from the lists since we are not officially saying profile anymore

Reply:
Yes, this code will be moved to test cases